CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Unreleased
 ----------
+
+21.10.0 (January 24, 2023)
```

Comment:
Year should be 2024

Reply:
That's what happens when you copy/paste without checking 😅 Updated!